doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -525,12 +531,13 @@ counterparts when the relation involved is not Leibniz equality.
 Notice, however, that using the prefixed tactics it is possible to
 pass additional arguments such as ``using relation``.
 
-.. tacv:: setoid_reflexivity
-          setoid_symmetry {? in @ident}
-          setoid_transitivity
-          setoid_rewrite {? @orientation} @term {? at @occurrences} {? in @ident}
-          setoid_replace @term with @term {? using relation @term} {? in @ident} {? by @ltac_expr3}
-   :name: setoid_reflexivity; setoid_symmetry; setoid_transitivity; setoid_rewrite; setoid_replace
+.. tacn:: setoid_reflexivity
+          setoid_symmetry {? in @ident }
+          setoid_transitivity @one_term
+          setoid_rewrite {? {| -> | <- } } @one_term {? with @bindings } {? at @occurrences } {? in @ident }
```

Comment:
`bindings` is not defined, but `bindings_list` is and means the same thing. Can you use the latter instead (and remove `bindings` from the `nitpick_ignore` list)?

Reply:
I don't think I should.  "bindings_list" isn't used in the mlg files.  All the uses of "bindings_list" are in the tactics chapter, which hasn't been updated.  If you like, I could rename "bindings_list" in the tactics chapter, I think that won't have ripple effects.

Reply:
Yes, renaming `bindings_list` into `bindings` everywhere in the tactics chapter is also fine to me.